backend/services/interfaces/classService.ts

```diff
@@ -74,7 +82,7 @@ export interface IClassService {
    */
   createStudent(
     student: StudentRequestDTO,
-    classId: string,
+    id: string,
```

Comment:
i think `id` might be confused with `studentId` instead of class? but if you think this is more clear then we can make the change!